rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -175,6 +175,36 @@ object ProcNormalizeMatcher {
           dataResults._2.knownFree)
       }
 
+      case p: PContr => {
+        import scala.collection.JavaConverters._
+        // A free variable can only be used once in any of the parameters.
+        // And we start with the empty free variable map because these free
+        // variables aren't free in the surrounding context: they're binders.
```

Comment:
Is it accurate to say that they're binders that are going to now be shadowed?

Reply:
No, they're binders that are going to shadow.

Reply:
Ah yes that is what I meant.